testnet/stacks-node/src/config.rs

```diff
@@ -273,6 +276,48 @@ impl ConfigFile {
         }
     }
 
+    pub fn mainnet() -> ConfigFile {
+        let burnchain = BurnchainConfigFile {
+            mode: Some("mainnet".to_string()),
+            rpc_port: Some(8332),
+            peer_port: Some(8333),
+            peer_host: Some("bitcoind.blockstack.org".to_string()),
+            ..BurnchainConfigFile::default()
+        };
+
+        let node = NodeConfigFile {
+            bootstrap_node: Some("047435c194e9b01b3d7f7a2802d6684a3af68d05bbf4ec8f17021980d777691f1d51651f7f1d566532c804da506c117bbf79ad62eea81213ba58f8808b4d9504ad@mainnet.blockstack.org:20444".to_string()),
+            miner: Some(false),
+            ..NodeConfigFile::default()
+        };
+
+        let balances = vec![
```

Comment:
If this is the function for booting the node to use mainnet, then shouldn't this be an empty vector?  I.e. all STX tokens should come from the genesis block, right?

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2173/commits/dbe8f0a60efc8d6cb6844d466422018847724ef3